Gordon360/Services/ProfileService.cs

```diff
@@ -88,6 +88,29 @@ public MailboxViewModel GetMailboxCombination(string username)
             return combo;
         }
 
+        /// <summary>
+        /// get the current user's birthday
+        /// </summary>
+        /// <param name="username">The current user's username</param>
```

Comment:
Technically, these are inaccurate comments. The username can be for any user and the birthdate returned is for whoever's username was passed in. That will be important in the future if we add a way for people to get other people's birthdates.